src/components/notifications/Notifications.jsx

```diff
@@ -0,0 +1,139 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import moment from 'moment'
+import { Alert, Avatar, Badge, Button, Popover } from 'antd'
+
+import './style.css'
+
+class Notifications extends Component {
+  constructor(props) {
+    super(props)
+    this.markAllAsRead = this.markAllAsRead.bind(this)
+    this.setVisibility = this.setVisibility.bind(this)
+    this.state = {
+      visible: false,
+    }
+  }
+
+  markAllAsRead() {
+    this.props.notifications.filter(notification => !notification.isRead).forEach((notification) => {
+      this.props.markNotificationAsRead(notification.id)
+    })
+    this.setVisibility(false)
+  }
+
+  setVisibility(visible) {
+    this.setState({ visible })
+  }
+
+  goToBoard(notification) {
+    this.props.markNotificationAsRead(notification.id)
+    window.location.replace(`/boards/${notification.board.id}`)
+  }
+
+  renderHeader() {
+    return (
+      <div className="notificationsHeader">
+        <h3>Notifications</h3>
+        <Button
+          className="notificationsMarkAllAsRead"
+          type={this.props.notifications.filter(notification => !notification.isRead).length > 0 ? 'primary' : 'default'}
+          icon="check"
+          size="small"
+          onClick={this.markAllAsRead}
+        >
+          Mark all as read
+        </Button>
+      </div>
+    )
+  }
+
+  renderContent() {
+    return this.props.notifications.length > 0 ? (
+      <div className="notificationsList">
+        {this.props.notifications
+          .filter(notification => !notification.isRead)
+          .sort((a, b) => moment(b.occuredAt) - moment(a.occuredAt))
```

Comment:
why sort on a difference and not on : 
.sort((a, b) => moment(b.occuredAt) > moment(a.occuredAt))

Reply:
Sort accept only integer to arrange according to the sign (negative / positive)